src/renderer/redux/modules/services/thunks.js

```diff
@@ -28,7 +28,7 @@ export function getServiceChildren(taxId) {
   return (dispatch, getState) => {
```

Comment:
wondering if you can add a specific call for getting all food successors as well. https://mofb-api.appspot.com/api/v2/taxonomy/food 
Of course that will force us to change it from being generic to be a specific call for 'food' successors (all children).

Reply:
I can but I don't see the value gain for doing that? Isn't this going to be for all services in the future?

Reply:
actually true although right now we dont have all services showing up under food. So going back to keeping interface simple in mvp (not including multilevels in filters like http://www.ohgo.com), sure in future. 

![image](https://user-images.githubusercontent.com/1335655/53142772-1f058280-3563-11e9-817d-3653fa5964c8.png)
we only showing 5 vs displaying all under https://mofb-api.appspot.com/api/v2/taxonomy/food

Reply:
so i could make this specific to food however that would require a rework of the drawer since currently i am filtering what is being rendered under each section (ie: "Food") based off of the taxId. So the taxId for the Food service is 11.. however when I query the /taxonomy/food endpoint I get entries that have a parentId of 12 (which is not the same as the taxId of "Food" I have gotten from the services call.

Reply:
right its bringing all successors under "food" including different levels that has different taxonomy id's.  e.g. 

https://mofb-api.appspot.com/api/v2/taxonomy/11/children (parent)
https://mofb-api.appspot.com/api/v2/taxonomy/12/children (child)
https://mofb-api.appspot.com/api/v2/taxonomy/17/children  (subchild)

Reply:
ok it should now handle this case where a service has children at different levels.
I changed how i store the data in redux from an array to an object.